cmd/api.go

```diff
@@ -192,6 +193,29 @@ func queryBalances(humanAddress string) (srv.ResultGetPegnetBalances, error) {
 	return res, nil
 }
 
+var issuance = &cobra.Command{
+	Use:              "issuance",
+	Short:            "Fetch the current issuance of all assets",
+	PersistentPreRun: always,
+	PreRun:           SoftReadConfig,
+	Run: func(cmd *cobra.Command, args []string) {
+		cl := srv.NewClient()
+		cl.PegnetdServer = viper.GetString(config.Pegnetd)
+		var res srv.ResultPTickerUintMap
+		err := cl.Request("get-pegnet-issuance", nil, &res)
+		if err != nil {
+			fmt.Printf("Failed to make RPC request\nDetails:\n%v\n", err)
+			os.Exit(1)
+		}
+
+		data, err := json.Marshal(res)
```

Comment:
We should print balances as their human readable forms. Not the `deca-nano` form. See balances